label_studio/projects/api.py

```diff
@@ -187,7 +187,8 @@ class ProjectAPI(generics.RetrieveUpdateDestroyAPIView):
     redirect_kwarg = 'pk'
 
     def get_queryset(self):
-        return Project.objects.with_counts().filter(organization=self.request.user.active_organization)
+        fields = get_fields_from_request(self.request)
```

Comment:
Why didn't you get query_param using using serializer?

Reply:
Fixed.